test/Microsoft.AspNetCore.Mvc.ApiExplorer.Test/DefaultApiDescriptionProviderTest.cs

```diff
@@ -216,6 +216,24 @@ public void GetApiDescription_CreatesMultipleDescriptionsForMultipleHttpMethods(
             }
         }
 
+        [Fact]
+        public void GetApiDescription_ParameterDescription_IncludesParameterDescriptor()
+        {
+            // Arrange
+            var action = CreateActionDescriptor(nameof(FromBody));
+
+            var expected = new BindingSource("Body", displayName: null, isGreedy: false, isFromRequest: false);
+
+            // Act
+            var descriptions = GetApiDescriptions(action);
+
+            // Assert
+            var description = Assert.Single(descriptions);
+            var parameters = description.ParameterDescriptions;
+            var actionParameterDescriptor = Assert.Single(action.Parameters);
+            var id = Assert.Single(parameters, p => p.ParameterDescriptor == actionParameterDescriptor);
```

Comment:
Get rid of the `var id = ` part.

And, there's only one parameter anywhere. So
``` c#
var parameterDescription = Assert.Single(description.ParameterDescriptions);
var actionParameterDescriptor = Assert.Single(action.Parameters);
Assert.Equal(actionParameterDescriptor, parameterDescription.ParameterDescriptor);
```